packages/neutrino/src/api.js

```diff
@@ -66,6 +67,14 @@ const getOptions = (options = {}) => {
       set(value) {
         entry = defaultTo('index', value);
       }
+    },
+    debug: {
```

Comment:
This section isn't needed. We have the getter/setters for the path options since we need to normalize the paths when they are overridden. `debug` isn't a path value, only a Boolean, so this can be removed.